SEED-platform/seed
# Inventory detail page - remove highlights for empty extra_data rows

#### Any background context you want to provide?
See issue

#### What's this PR do?
Fix bug that was highlighting empty extra data columns on inventory detail pages. Some historical states had extra_data information completely missing, while other were present but set to null. This PR normalizes the missing to be null.

#### How should this be manually tested?
View inventory detail page with empty extra data columns and see that they aren't highlighted.

#### What are the relevant tickets?
#1591 

#### Screenshots (if appropriate)
<img width="1394" alt="Screen Shot 2020-02-10 at 9 18 21 PM" src="https://user-images.githubusercontent.com/30608004/74211208-21695c80-4c4c-11ea-9cb4-fa879ee308d6.png">
